cluster-provision/common-scripts/config-cni.sh

```diff
@@ -1,21 +1,16 @@
-#! /bin/bash
+#!/bin/env bash
 
 set -e
 
-function configure_calico(){
-    cat <<EOF >/etc/NetworkManager/conf.d/calico.conf
-[keyfile]
-unmanaged-devices=interface-name:cali*;interface-name:tunl*
-EOF
-
-    sysctl -w net.netfilter.nf_conntrack_max=1000000
-    echo "net.netfilter.nf_conntrack_max=1000000" >> /etc/sysctl.conf
-
-    systemctl restart NetworkManager
-}
+cni_name=$1
+script_d=$cni_name/script.d
+default_cidr="192.168.0.0/16"
```

Comment:
maybe those should be exported in order to be valid on other scrips?

Reply:
We source this scripts so they are already exported.